recipes/newrelic/infrastructure/ohi/kafka/windows.yml

```diff
@@ -0,0 +1,53 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: kafka-open-source-integration
+displayName: kafka Open Source Integration
+description: New Relic install recipe for default kafka Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-kafka
+
+dependencies:
+  - infrastructure-agent-installer
+
+installTargets:
+  - type: host
+    os: windows
+    kernelArch: x86_64
+
+keywords:
+  - Infrastructure
+  - Integration
+  - kafka
+
+# Kafka runs under java process in windows, so we checking if kafka port 9092 is running under Java
+processMatch:
+  - kafka
```

Comment:
@NRhzhao could we also validate on java? Maybe something like java\.exe*.kafka (double check the regex)

Reply:
@Julien4218  I tried java*kafka, and it doesn't seem to be working.   Should I dig into this deeper?

![image](https://user-images.githubusercontent.com/94071514/146419521-6c872cf8-d520-43f2-8085-375b09ed3ef1.png)